slam/package.xml

```diff
@@ -49,19 +49,17 @@
   <!-- Use doc_depend for packages you need only for building documentation: -->
   <!--   <doc_depend>doxygen</doc_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>nav_msgs</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>rospy</build_depend>
   <build_depend>std_msgs</build_depend>
-  <build_export_depend>nav_msgs</build_export_depend>
   <build_export_depend>roscpp</build_export_depend>
   <build_export_depend>rospy</build_export_depend>
   <build_export_depend>std_msgs</build_export_depend>
-  <exec_depend>nav_msgs</exec_depend>
```

Comment:
same as above, this seems a tiny bit sus since nav_msgs includes Odometry, which I think Slam uses in the `slam/robot/state` topic, but as long as it re-builds from clean it should be fine

Reply:
We haven't added publishing and subscribing yet. We are planning to add this feature soon.